docs/source/user-guide/vision/quickstarts/plot_classification_tutorial.py

```diff
@@ -99,21 +83,12 @@ def __getitem__(self, index: int):
 
     def get_cv2_image(self, image):
         if isinstance(image, PIL.Image.Image):
-            image_np = np.array(image).astype('uint8')
-            return image_np
+            return np.array(image).astype('uint8')
         elif isinstance(image, np.ndarray):
             return image
         else:
             raise RuntimeError("Only PIL.Image and CV2 loaders currently supported!")
 
-# Just normalization for validation
-data_transforms = transforms.Compose([
-        transforms.Resize(256),
-        transforms.CenterCrop(224),
-        transforms.ToTensor(),
-        transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
-    ])
-
 data_dir = 'hymenoptera_data'
 # Just normalization for validation
 data_transforms = A.Compose([
```

Comment:
this can be inside the AntsBeesDataset and save adding it afterwards

Reply:
The datasets receive 2 different transform lists

Reply:
oh ok